#An array of 2D points

knowledge_server/Point2D[] points